2021/43xxx/CVE-2021-43085.json

```diff
@@ -2,30 +2,7 @@
     "CVE_data_meta": {
```

Comment:
CNA(s) of user (apache|openssl) do not match owning CNA.

Reply:
Please have owning CNA reassign this CVE to OpenSSL CNA due to breach of CNA guidelines.